src/argilla_llama_index/llama_index_handler.py

```diff
@@ -317,15 +295,22 @@ def _add_missing_metadata_properties(
                 metadata_name = "total_time"
             required_metadata_properties.append(metadata_name)
 
-        existing_metadata_properties = [property.name for property in dataset.metadata_properties]
-        missing_metadata_properties = [property for property in required_metadata_properties if property not in existing_metadata_properties]
+        existing_metadata_properties = [
+            property.name for property in dataset.metadata_properties
+        ]
+        missing_metadata_properties = [
+            property
+            for property in required_metadata_properties
+            if property not in existing_metadata_properties
+        ]
 
         for property in missing_metadata_properties:
-            title= " ".join([word.capitalize() for word in property.split('_')])
+            title = " ".join([word.capitalize() for word in property.split("_")])
             if title == "Llm Time":
```

Comment:
perhaps we can add a `lower` here?

Reply:
Right now we have the first letter in uppercase. Shall we go for all in lowercase. If so, I'd remove the lines below as well. But I'd vote to maintain the `capitalize()` instead of the `lower()`